### PR TITLE
RANCHER-1907: add integration with Kafka

### PR DESCRIPTION
**Pull request by yaroslavishchenko**
Related Jira ticket:
[RANCHER-1907:] Add kafka support for mod-invoice-storage, mod-organizations-storage(https://folio-org.atlassian.net/browse/RANCHER-1907)

### CODE REVIEW

File: charts/mod-invoice-storage/values.yaml [hunk 1]
```diff
@@ -164,6 +164,12 @@ integrations:
     querytimeout: 60000
     maxpoolsize: 5
     existingSecret: ""
+  
+  kafka:
+    enabled: true
```
Comment (epam-avramenko) [inline charts/mod-invoice-storage/values.yaml:169]: Please set enabled: false by default.

File: charts/mod-organizations-storage/values.yaml [hunk 2]
```diff
@@ -164,6 +164,12 @@ integrations:
     querytimeout: 60000
     maxpoolsize: 5
     existingSecret: ""
+  
+  kafka:
+    enabled: true
```
Comment (epam-avramenko) [inline charts/mod-organizations-storage/values.yaml:169]: Please set enabled: false by default.

**Review by epam-avramenko — CHANGES_REQUESTED** [review]

File: charts/mod-organizations-storage/values.yaml (hunk 2 above)
Comment (OHaimanov) [reply charts/mod-organizations-storage/values.yaml:169]: ++

File: charts/mod-invoice-storage/values.yaml (hunk 1 above)
Comment (OHaimanov) [reply charts/mod-invoice-storage/values.yaml:169]: ++

**Review by epam-avramenko — APPROVED** [review]

File: charts/mod-invoice-storage/values.yaml (hunk 1 above)
Comment (yaroslavishchenko) [reply charts/mod-invoice-storage/values.yaml]: @epam-avramenko
Adjusted accordingly to your recommendation.

**Review by OHaimanov — APPROVED** [review]

**Pull request merged**